table.csv

```diff
@@ -35,6 +35,7 @@ multihashes,,
 sha1,               ,                         0x11
 sha2-256,           ,                         0x12
 sha2-512,           ,                         0x13
+dbl-sha2-256,       ,                         0x56
```

Comment:
Yeah, adding a notion of 'rounds' to multihash really opens the door to a lot of complexity. You could hash, truncate, hash, or hash, append salt, hash, etc